pkg/storage/filesystem/btrfs_ci_test.go

```diff
@@ -34,20 +34,26 @@ var (
 
 type TestDevices map[string]string
 
-func (d TestDevices) Loops() Devices {
+func (d TestDevices) Loops(t *testing.T) Devices {
 	mgr := lsblkDeviceManager{
 		executer: executerFunc(run),
 	}
-	for _, loop := range d {
+	for loopTempPath, loop := range d {
+		size, err := FilesUsage(loopTempPath)
+		if err != nil {
+			require.NoError(t, err)
+		}
 		mgr.cache = append(mgr.cache, DeviceInfo{
 			Path: loop,
 			Rota: false,
+			Size: size,
 		})
+
 	}
 
 	devices, err := mgr.Devices(context.Background())
 	if err != nil {
-		panic(err)
+		require.NoError(t, err)
 	}
```

Comment:
you should remove the error check, replace this with only `require.NoError(t, err)`